.github/workflows/welcome-external-pr.yml

```
@@ -0,0 +1,65 @@
+name: External contributor greeter
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - reopened
+
+env:
+  ORGANIZATION: Ethereum
+  DRY_RUN: true
+
+jobs:
+  comment-external-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get organization members
+        id: get_members
+        env:
+          GH_TOKEN: ${{ secrets.READ_ORG }}
+          CONTRIBUTOR: ${{ github.event.pull_request.user.login }}
+        run: |
+          gh api graphql \
+            --raw-field organization="$ORGANIZATION" \
+            --raw-field query='
+              query($organization: String!, $cursor: String) {
+                organization(login: $organization) {
+                  membersWithRole(first: 100, after: $cursor) {
+                    pageInfo {
+                      hasNextPage,
+                      endCursor
+                    }
+                    nodes {
+                      login
+                    }
+                  }
+                }
+              }' > org_members.json
+          echo "CONTRIBUTOR_IS_ORG_MEMBER=$(
+            jq \
+              --arg contributor $CONTRIBUTOR \
+              '.data.organization.membersWithRole | any(.nodes[].login; . == $contributor)' \
+              org_members.json
+          )" >> $GITHUB_OUTPUT
+
+      - name: Comment on external contribution PR
+        if: ${{ steps.get_members.outputs.CONTRIBUTOR_IS_ORG_MEMBER == 'false' }}
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PR: ${{ github.event.pull_request.html_url }}
+        run: |
+          echo "Commenting in a newly submitted or reopened external PR: $PR"
+          if [[ $DRY_RUN == 'false' ]]; then
+            gh pr edit "$PR" --add-label "external contribution :star:"
+            comment_body=(
+              "Thank you for your contribution to the Solidity compiler! A team member will follow up shortly."
+              "\n\n"
+              "If you haven't read our [contributing guidelines](https://docs.soliditylang.org/en/latest/contributing.html) and our [review checklist](https://github.com/ethereum/solidity/blob/develop/ReviewChecklist.md) before, "
```

The point of this was to be able to wrap lines for readability, so I'd still do this before merging:
```suggestion
              "If you haven't read our [contributing guidelines](https://docs.soliditylang.org/en/latest/contributing.html) and our "
              "[review checklist](https://github.com/ethereum/solidity/blob/develop/ReviewChecklist.md) before, "
```